### DISCUSSION TITLE
[Feat]: VUE documentation // show examples on how to use the scrollNext, slideTo and scrollPrev methods

### DISCUSSION THREAD

**Discussion by Starlette** [davidjerleke/embla-carousel#1018] [Ideas] [answered]
### Which variants of Embla Carousel are relevant to this feature request?

- [ ] embla-carousel **(Core)**
- [ ] embla-carousel-react
- [X] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

### Feature description

Currently the documentation only shows you how to use it in React or Vanilla JS. I'm struggling to get it working in Vue. The documentation on Vue is only one page with the bare minimum examples. Can we have more advanced examples or maybe add Vue as part of the generator?

### Before submitting

- [X] I've made research efforts and searched the documentation
- [X] I've searched for existing issues
- [X] I agree to follow this project's Contributing Guidelines for feature requests

**Comment by davidjerleke** [comment]
@Starlette thanks for your feature request.

Yes, I think this should be added to the docs. More specifically [here](https://www.embla-carousel.com/guides/previous-and-next-buttons/), and not only for Vue but for all supported frameworks using tabs like [this section](https://www.embla-carousel.com/api/options/#constructor-options).

Unfortunately, I don't have much time so I don't know if I will be able to do this myself.

**Reply by davidjerleke** [reply, marked as answer]
@Starlette There's now a [section in the docs](https://www.embla-carousel.com/guides/previous-and-next-buttons/#with-vue) that demonstrates how to do this with Vue.
